assignment/resume-bootstrap/resume.html

```diff
@@ -0,0 +1,101 @@
+<html>
+    <head>
+        <title>Sarah-Yacoub-Resume</title>
+        <link type="text/css" rel="stylesheet" href="bootstrap.min.css"/>
+        <link type="text/css" rel="stylesheet" href="resume-CSS.css"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        
+    </head>
+    <body>
+            <section id="maindiv" class="container">
+                        <header id="header">
+                             <div class="row">
+                                 <div class="col-md-4 col-sm-6 col-xs-12">
```

Comment:
Why do you need this parent `<div>`? You can apply the bootstrap classes on the `<section>` directly.